apps/choleskey/cholesky_tiled.cpp

```diff
@@ -0,0 +1,192 @@
+#include <bits/stdc++.h>
+#include <cblas.h>
+#include <lapacke.h>
+#include <omp.h>
+#include <algorithm>
+#include <cmath>
+#include <cstdlib>
+#include <experimental/linalg>
+#include <iostream>
+#include <utility>
+#include <vector>
+#include "help.hpp"
+
+using data_type = double;
+
+//#include "time.hpp"
+#define SWITCH_CHAR '-'
```

Comment:
what is SWITCH_CHAR and has it been used?

Reply:
> what is SWITCH_CHAR and has it been used?

yes， forgot to remove this definition